### ISSUE TITLE
Wrong end time when resizing an event to the next day

### ISSUE THREAD

**Issue by flack** [fullcalendar/fullcalendar#3038]
Steps to reproduce:

1: go to http://fullcalendar.io/js/fullcalendar-2.6.0/demos/selectable.html
2: go to week view
3: select a timespan from Monday to Tuesday

Expected result:
- selected end time is shown on fc-event-helper

Actual result:
- 12:00 is shown, no matter when the event actually ends

Screenshot:

![wrong_end_time](https://cloud.githubusercontent.com/assets/425166/12899101/2940c772-ceb0-11e5-9017-1a8151c98055.png)

**Comment by arshaw** [comment]
this was intentional behavior for events that span across the nextDayThreshold. If you scroll up you'll see the event on Monday continue from midnight to 9am on Tuesday.

If there is enough demand, we could potentially introduce a setting for fine-grain control of this behavior.

**Comment by flack** [comment]
hmm, ok, but if it's intentional, then why do I get a different end time in `event.end` when the `eventDrop` callback runs? In the screenshot above, `event.end` will be 9:00 AM, it's only the displayed value on the helper that says 12:00

**Comment by arshaw** [comment]
this issue concerns itself with how an event's text is displayed. concerned with what text renders on each event segment when it spans across two days. It's merely what text is displayed. the underlying data, the `event.end` should be as expected.

**Comment by solanaprojordan** [comment]
I, too, would like this functionality. It would be nice to have the option of whether to show the splits or the actual event time on both segments.

**Comment by jayatulsiani** [comment]
yes i would like this functionality to be implemented. can it be done using range option? i tried using range but in vain. can somebody help me here

**Comment by bartvdklu** [comment]
It would be nice to have this functionality! Hope it gets implemented. 👍

**Comment by blissdev** [comment]
This would be very useful for my team's usage.

**Comment by ekrismer** [comment]
I've identified at least three locations that are affected by this issue.

Have a look at this Codepen: https://codepen.io/erabl/pen/GRXgNVm

1. Multiday events that exceed the current view's date range
![grafik](https://user-images.githubusercontent.com/8388464/219154013-02c511d4-c25b-45bb-bc57-a626217fbaa2.png)

2. Multiday events that are part of a '+ more' popover
![grafik](https://user-images.githubusercontent.com/8388464/219156593-eab4bdc8-53ff-4171-a389-fe70a3b6b647.png)

3. Multiday events in a `timeGrid`
![grafik](https://user-images.githubusercontent.com/8388464/219160398-3815bee4-747e-415f-8848-41ddc49d6fe2.png)

I was able to find a workaround for `v5.11.0` ([Codepen](https://codepen.io/erabl/pen/ExeamRM)) using 
```javascript
eventContent: (arg) => {
  if (arg.event.start && arg.event.end) {
    // this is just a quick hack: use a proper date formatter with timezone!
    arg.timeText = `${arg.event.start.toISOString().slice(11,16)} - ${arg.event.end.toISOString().slice(11,16)}`;
  }
}
```
but could not get this to work with `v6.1.4`.

@arshaw, @acerix do you know a workaround that is compatible with v6?

Are there any plans to add a flag that controls how multiday event times are rendered?

Possibly related issues: #6936, #3765, #5826.

**Comment by ekrismer** [comment]
I figured out a workaround for `v6`: Content Injection now requires to `return true` to render the default content (see https://fullcalendar.io/docs/upgrading-from-v5#content-injection).

```javascript
eventContent: (arg) => {
  if (arg.event.start && arg.event.end) {
    // this is just a quick hack: use a proper date formatter with timezone!
    arg.timeText = `${arg.event.start.toISOString().slice(11,16)} - ${arg.event.end.toISOString().slice(11,16)}`;
  }
  return true; // render default content
}
```

This hint unfortunately is not present in the regular documentation: https://fullcalendar.io/docs/content-injection
